src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,11 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
+        if (userService.findByEmail(email) != null) {
+            return userService.findByEmail(email).getPassword().equals(password);
+        }
         return false;
```

Comment:
Please try to write this method using one return 
`return` user != null && user.pasword.equals(password)`